README.md

```diff
@@ -1,3 +1,131 @@
 # aldryn project template
 
-See feature/prototype branch for first prototype
+Welcome to the first prototype of the new aldryn project template.
+
+**Use https://github.com/aldryn/prototype-base-project-meta to setup locally.**
+
+Your feedback is very valuable! Please help.
+
+This prototype is intended to get feedback in order to flesh out a good api
+and very understandable project structure. Add github issues on this repo for
+questions, ideas, bugs. Please also look at existing issues of other people and
+join the discussion.
+
+So... what am I looking at?
+
+This repo represents a user-accessible git repo of a website on aldryn. It's
+goal is to have as little non-project specific code as possible. All the heavy
+lifting is outsourced into Addons.
+
+```
+addons   # all Addons installed on this website
```

Comment:
No. This would contain the following files in git version history:
 - ``addon.json``: copied in by the controlpanl each time the addon is updated
 - ``aldryn_config.py``: copied in by the controlpanl each time the addon is updated
 - ``settings.json``: all non secret settings of the addon (will automatically be updated by the controlpanel when changes are made over the UI)

In a later iteration of the api, we could have addon authors put ``aldryn_config.py`` inside their python package. Then it would not have to be copied into this directory and could be read directly from the installed package from site-packages. ``addon.json`` could probably be made obsolete, since the only information it holds it the "package name" used on aldryn. And that is identical to the pypi package name in 99% of the cases.